changelogs/fragments/602-show-all-slaves-status.yaml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - mysql_info - The `slave_status` filter was returning an empty list on MariaDB with multiple replication channels. It now returns all channels by running `SHOW ALL SLAVES STATUS` for MariaDB servers. (https://github.com/ansible-collections/community.mysql/issues/603).
```

```suggestion
  - mysql_info - the ``slave_status`` filter was returning an empty list on MariaDB with multiple replication channels. It now returns all channels by running ``SHOW ALL SLAVES STATUS`` for MariaDB servers (https://github.com/ansible-collections/community.mysql/issues/603).
```
to make the changelog entries look consistent